x/stake/types/inflation_test.go

```diff
@@ -82,41 +82,6 @@ func TestProcessProvisions(t *testing.T) {
 	checkFinalPoolValues(t, pool, sdk.NewRat(initialTotalTokens), cumulativeExpProvs)
 }
 
-// Benchmark precision
-func BenchmarkProcessProvisions10000(b *testing.B) {
```

Comment:
Could this be easily adapted to just a benchmark of the inflation calculation? I think thats valuable, regardless of whether or not we're doing rational math.

Reply:
But yeah I mean the inflation calculation is within here, we could benchmark both, but honestly think it's probably enough to benchmark processing provisions because it's so tightly coupled with the inflation calculation (and is the higher level calculation)

Reply:
Sounds good! Thanks for looking into it!